### ISSUE TITLE
Training a CNN model for emotion classification

### ISSUE THREAD

**Issue by mateuszb0k** [mateuszb0k/humanoid-head#7]
Building and training a custom CNN architecture in TensorFlow for face recognition and a CNN for emotional state estimation.
Network architecture defined
training process completed in an isolated environment
Validation Accuracy achieved at a minimum of 50%.
Trained model exported/saved
